forge-gui/res/cardsfolder/upcoming/quintorius_loremaster.txt

```diff
@@ -0,0 +1,15 @@
+Name:Quintorius, Loremaster
+ManaCost:3 R W
+Types:Legendary Creature Elephant Cleric
+PT:3/5
+K:Vigilance
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigChangeZone | TriggerDescription$ At the beginning of your end step, exile target noncreature, nonland card from your graveyard. Create a 3/2 red and white Spirit creature token.
+SVar:TrigChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | ValidTgts$ Card.nonCreature+nonLand+YouOwn | TgtPrompt$ Select target noncreature, nonland card in your graveyard | RememberTargets$ True | SubAbility$ DBToken
```

Comment:
Don't need Remembering for such cases anymore

Reply:
Reworked the effect to use Targeted. Thanks!